### PR TITLE
Fixes #15101: the "disabled" label next to rule name isn't present anymore

### PR DESCRIPTION
**Pull request by RaphaelGauthier**
https://issues.rudder.io/issues/15101

### CODE REVIEW

File: webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/components/RuleGrid.scala [hunk 1]
```diff
@@ -634,12 +634,10 @@ class RuleGrid(
     TimingDebugLogger.trace(s"Rule grid: transforming into data: get rule data: checkbox callback: ${t2-t1}ms")
     // Css to add to the whole line
     val cssClass = {
-      val disabled = if (line.rule.isEnabled) {
-        ""
-      } else if(line.rule.isEnabledStatus){
-        "unappliedRule"
```
Comment (VinceMacBuche) [inline webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/components/RuleGrid.scala]: You should not remove unappliedRule state, it's not the same then disabled

**Comment by RaphaelGauthier** [comment]
Commit modified

**Comment by RaphaelGauthier** [comment]
Commit modified

**Comment by RaphaelGauthier** [comment]
Commit modified

**Comment by RaphaelGauthier** [comment]
Commit modified

**Pull request merged**